baselines/requireJquery.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const requireJquery: string;
```

Comment:
It doesn't seem that `generateIdentifierDeclarationFile` is generating the correct files. However, that one is not related to this PR.